metadata.yaml

```
@@ -9,6 +9,12 @@ summary: A MongoDB operator charm
 series:
   - focal
 
+provides:
+  database:
+    interface: database-client
```

This should be changed to be specific to MongoDB.

```suggestion
    interface: mongodb-client
```